### PR TITLE
LPS-158682 Migrate Runs Table from GraphQL to Rest

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/Run.tsx [hunk 1]
```diff
@@ -92,7 +61,7 @@ const Runs = () => {
 						},
 					],
 				}}
-				transformData={transformData}
+				transformData = {getRunTransformData}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/Run.tsx:64]: Please, remove these spaces.
```suggestion
				transformData={getRunTransformData}
```

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/services/rest/TestrayRun.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	externalReferencePK: string;
+	externalReferenceType: string;
+	id: number;
+	jenkinsJobKey: string;
+	name: string;
+	number: string;
+	status: string;
+};
+
+const runsResource = '/runs';
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/services/rest/TestrayRun.ts:33]: Please, remove this line and put this value in this file [/Run.tsx](https://github.com/pt-liferay-solutions/liferay-portal/pull/1679/files#diff-3da1171b0efc0d6fd85e267123946f392a9b28171f571cb3623a47a2a2fd2f21) resource property directly.

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :+1: But needs some adjustments.

**Review by welitonpe — APPROVED** [review]
LGTM

**Comment by welitonpe** [comment]
Just starting reviewing :)

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
LGTM :1st_place_medal:  Thanks :smile:

**Comment by kevenleone** [comment]
resent: https://github.com/pt-liferay-solutions/liferay-portal/pull/1755

**Pull request closed without merging**